File: 58-length-of-last-word/length-of-last-word.js
/**
 * @param {string} s
 * @return {number}
 */
var lengthOfLastWord = function(s) {
       let res= s.split(" ").filter(n=>n!=="");
      return res[res.length-1].length
};
console.log(lengthOfLastWord("Hello World"))
console.log(lengthOfLastWord("   fly me   to   the moon  "))
console.log(lengthOfLastWord("luffy is still joyboy"))

File: 58-length-of-last-word/length-of-last-word.py
class Solution(object):
    def lengthOfLastWord(self, s):
        res=s.strip(" ")
        res1=res.split(" ")
        return len(res1[-1])
obj=Solution()
print(obj.lengthOfLastWord("Hello World"))
print(obj.lengthOfLastWord("   fly me   to   the moon  "))
print(obj.lengthOfLastWord("luffy is still joyboy"))